test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -280,115 +281,29 @@ $ kafka-verify format=avro sink=materialize.public.compaction_test_sink sort-mes
-    CONSISTENCY (TOPIC 'consistency-default-avro')
-  FORMAT JSON
-contains:CONSISTENCY FORMAT JSON not yet supported
-
 > CREATE SINK double_avro FROM simple_view
   INTO KAFKA CONNECTION kafka_conn TOPIC 'double-avro'
-    CONSISTENCY (TOPIC 'consistency-double-avro' FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn)
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn
 
 $ kafka-verify format=avro sink=materialize.public.double_avro sort-messages=true
 {"before": null, "after": {"row": {"a": 1, "b": 2, "c": 3}}, "transaction": {"id": "<TIMESTAMP>"}}
 
 > CREATE SINK json_avro FROM simple_view
   INTO KAFKA CONNECTION kafka_conn TOPIC 'json-avro'
-    CONSISTENCY (TOPIC 'consistency-json-avro' FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn)
   FORMAT JSON
 
 $ kafka-verify format=json sink=materialize.public.json_avro sort-messages=true key=false
 {"before": null, "after": {"a": 1, "b": 2, "c": 3}, "transaction": {"id": "<TIMESTAMP>"}}
 
-! CREATE SINK json_reuse_topic_no_parens FROM simple_view
-  INTO KAFKA CONNECTION kafka_conn TOPIC 'json-reuse-topic'
-  FORMAT JSON
-contains:For FORMAT JSON, you need to manually specify an Avro consistency topic
-
-> CREATE SINK json_reuse_topic_with_parens FROM rt_binding_consistency_test_source
-  INTO KAFKA CONNECTION kafka_conn TOPIC 'json-reuse-topic-2'
-    CONSISTENCY (TOPIC 'consistency-json-avro-2' FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn)
-  FORMAT JSON
-
-# Same as above!
-> CREATE SINK json_reuse_topic_with_parens_and_with FROM rt_binding_consistency_test_source
-  INTO KAFKA CONNECTION kafka_conn TOPIC 'json-reuse-topic-3'
-    CONSISTENCY (TOPIC 'consistency-json-avro-3' FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn)
-  FORMAT JSON
-
-# Default consistency format is Avro if consistency_topic is used
-> CREATE SINK default_avro_2 FROM simple_view
-  INTO KAFKA CONNECTION kafka_conn TOPIC 'default-avro-2'
-  CONSISTENCY (TOPIC 'default-consistency-avro')
-  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn
-
-$ kafka-verify format=avro sink=materialize.public.default_avro sort-messages=true
-{"before": null, "after": {"row": {"a": 1, "b": 2, "c": 3}}, "transaction": {"id": "<TIMESTAMP>"}}
-
 > CREATE SINK json_avro_upsert_key FROM simple_view
   INTO KAFKA CONNECTION kafka_conn TOPIC 'json-avro-upsert-key'
   KEY (b)
-    CONSISTENCY (TOPIC 'consistency-json-avro-upsert-key' FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn)
   FORMAT JSON
   ENVELOPE UPSERT
 
 $ kafka-verify format=json sink=materialize.public.json_avro_upsert_key key=true
 {"b": 2} {"a": 1, "b": 2, "c": 3, "transaction": {"id": "<TIMESTAMP>"}}
-
-# Verify compaction of exactly once sinks.
-
-# TODO: we've disabled this after observing it be flaky in CI. Re-enable once we've
-# fixed the issue.
-# $ verify-timestamp-compaction source=input_csv max-size=3 permit-progress=true
```

Comment:
hmm, has `verify-timestamp-compaction` just rotted away by now?

Reply:
It seems to have been commented out in `action.rs`, though the support code is still there. I'll have a look.